spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -157,7 +157,8 @@
       },
       "file": {
         "filename": generic_file.filename,
-        "content_path": generic_file_content_path
+        "content_path": generic_file_content_path,
+        "original_checksum": generic_file.original_checksum
```

Comment:
Could we have a test for the `original_checksum` actually? To make sure that the original_checksum is produced correctly.

Reply:
We actually did add this when we first added `#generic_file_content_path` as seen [here](https://github.com/galterlibrary/digital-repository/blob/a5feaf4b39835e1b3709b03a004380b2c30e9d7d/spec/models/invenio_rdm_record_converter_spec.rb#L277), but let me know if we want more edge cases than just one.

Reply:
I guess I thought the spec tests used an actual file: then we could have tested our expectation that `original_checksum` is the MD5 checksum of the actual file. It's fine if we don't since the test might not have been representative of production anyway and it's better to get feedback from production first.

Thanks!